force-app/main/default/classes/SOQL.cls

```diff
@@ -85,9 +103,17 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    public static Map<Id,Task> getTasksDueNextWeek () {
     //  The body of the method
-    //}
+        List<Task> allTasksDueIn7Days = [SELECT Id, WhatId, Subject, ActivityDate, Status, Priority, OwnerId, AccountId, isClosed FROM Task where ActivityDate > TODAY and  ActivityDate <= NEXT_N_DAYS:7];
```

Comment:
Great use of the date key words you have available in SOQL/SQL.

Be sure to look at vertical formatting.